suites/pacific/rbd/tier-3_rbd_cache_scenarios.yaml

```diff
@@ -0,0 +1,99 @@
+# Tier3: Extended RBD cache scenario tests.
```

Comment:
As discussed, please update polarion and test suite to tier-2.

Reply:
Done.